### PR TITLE
feat: build sub-packages by scope for individual publishing

### PR DESCRIPTION
**Pull request by uncenter**

### CODE REVIEW

File: src/build.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 const names = require('all-the-package-names');
 const fs = require('fs');
 const path = require('path');
+const fancyPackage = false;
```
Comment (uncenter) [inline src/build.js:4]: why is this up here? maybe make it an environment variable?

Comment (PatrickJS) [reply src/build.js:4]: haha fair

Comment (uncenter) [reply src/build.js:4]: process.env.PRETTY_PRINT_PACKAGE_JSON or smt

File: src/build.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+			acc[`@everything-registry/${curr}`] = require('../package.json').version;
+			return acc;
+		}, {}),
+	},
+	fancyPackage,
+);
+fs.writeFileSync(path.join(dir, 'package.json'), pkgJson);
+fs.writeFileSync(
+	path.join(dir, 'index.js'),
+	"console.log('You have installed everything but at what cost...');",
```
Comment (uncenter) [inline src/build.js:69]:
```suggestion
	"console.log('You have installed everything... but at what cost?');",
```

File: src/build.js (hunk 1 above)
Comment (PatrickJS) [reply src/build.js]:
```
process.env.FANCY_PACKAGE || false
```

**Comment by PatrickJS** [comment]
image

**Comment by uncenter** [comment]
Almost, lemme do some more improvements.

**Comment by uncenter** [comment]
That check script might not work on ubuntu... I think the `stat` command is different on ubuntu/macOS?

**Review by PatrickJS — APPROVED** [review]

**Pull request merged**